template_feed/Microsoft.DotNet.Common.ProjectTemplates.3.0/content/ClassLibrary-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": ["Common", "Library"],
+  "name": "Class library",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A project for creating a class library that targets .NET Standard or .NET Core",
+  "groupIdentity": "Microsoft.Common.Library",
+  "precedence": "4000",
```

Comment:
Think precedence needs to be bumped up - @seancpeters ?

Reply:
I'd say yes, precedence needs to be changed, 5000 perhaps? The 2.2 templates have precedence 4000. If the 3.0 templates have the same precedence as the 2.2 templates, we'll get strange behavior in the cli.